operator/pkg/gateway-api/cell.go

```diff
@@ -191,7 +191,7 @@ func checkCRD(ctx context.Context, clientset k8sClient.Clientset, gvk schema.Gro
 	return nil
 }
 
-func checkRequiredCRDs(ctx context.Context, clientset k8sClient.Clientset) error {
+func CheckRequiredCRDs(ctx context.Context, clientset k8sClient.Clientset, requiredGVK []schema.GroupVersionKind) error {
```

Comment:
I'd prefer to avoid exposing this functionality for other packages and introduce the dependency.

Either move this function to a helper package (`operator/watchers` ?) or duplicate it in the clustermesh/mcsapi cell (IMO wouldn't be that bad - at least better than the cross-reference.)

Reply:
Sure that's fair, thanks for the review :pray:.

I ended up duplicating the code as you also proposed as I am not sure we will have another package requiring third party CRDs and it's not clear what would be the best common package to get that (like `operator/watchers` seems a a bit odd imo).